package.json

```diff
@@ -25,20 +25,25 @@
     "test": "npm-run-all lint test:*",
     "test:ember": "ember test"
   },
+  "overrides": {
+    "n3": {
+      "readable-stream": "4.3.0"
+    }
+  },
   "dependencies": {
     "date-fns": "^2.29.2",
     "ember-cli-release": "^1.0.0-beta.2",
     "ember-fetch": "^8.1.1"
   },
   "devDependencies": {
-    "@appuniversum/ember-appuniversum": "^1.3.0",
```

Comment:
Hmmm, this might require more work than simply bumping. Appuniversum v2 removed some deprecated code, so it's possible that was in use here. Did you see any deprecations before bumping?

Reply:
Ok, I just quickly checked and it seems most were resolved already. Only 1 remained and I pushed a commit for that.